### ISSUE TITLE
Could not resolve "node:path" - meow package

### ISSUE THREAD

**Issue by davidwinter** [postui/esm.sh#100]
Hello! How should built-in node modules be handled? For example, trying to import the `meow` package, the following error happens:

```js
/* esm.sh - error */
throw new Error("[esm.sh] " + "Could not resolve \"node:path\"  (Imported by \"meow\")");
export default null;
```

https://esm.sh/meow

I was trying to use [import mapping](https://deno.land/manual@v1.13.1/npm_nodejs/import_maps) with Deno to remap this, but looks like the package doesn't even reach me in order for Deno to be able to do this.

**Comment by ije** [comment]
@davidwinter thanks, i will fix this asap!

**Comment by ije** [comment]
fixed in 021448e, pls tray again!

**Comment by davidwinter** [comment]
Thanks, @ije - that resolved that issue. Though this might be unrelated, but another error I now encounter as a result of the dependency tree of `meow` is the following, where an export isn't correct:

```
error: Uncaught SyntaxError: The requested module '/v45/path-browserify@1.0.1/deno/path-browserify.js' does not provide an export named 'resolve'
    at <anonymous> (https://cdn.esm.sh/v45/yargs-parser@20.2.9/deno/yargs-parser.js:1:343)
```
